container_engine/django_postgres_redis/Makefile

```diff
@@ -0,0 +1,71 @@
+GCLOUD_PROJECT:=$(shell gcloud config list project --format="value(core.project)")
+ZONE=$(shell gcloud config list compute/zone --format="value(compute.zone)")
+AUTOSCALE_GROUP=$(shell gcloud container clusters describe $(CLUSTER_NAME) --zone $(ZONE) --format yaml | grep -A 1 instanceGroupUrls | awk -F/ 'FNR ==2 {print $$NF}')
```

Comment:
This will fail if the cluster doesn't exist. It seems you only use it in one target, maybe it move it down there?